src/PKSim.Core/Snapshots/Mappers/MoleculeMapper.cs

```diff
@@ -49,6 +49,13 @@ public override async Task<Molecule> MapToSnapshot(IndividualMolecule molecule)
          return snapshot;
       }
 
+      protected override bool ShouldExportParameterToSnapshot(IParameter parameter)
```

Comment:
@Yuri05 I always export the global parameters. This is very easy as we already had an extension method checking if a parameter was one of the global parameters

Reply:
This works ok but not perfect. Upon reload from snapshot, it looks like so for a molecule that was renamed

![image](https://user-images.githubusercontent.com/1041237/74189722-e997f000-4c1e-11ea-8856-b0b682a8f320.png)

The default value is not saved in the snapshot. Thus when we reload from snapshot, default differs from current and it is shown as changed.  I am not sure if you have an idea how to circumvent this. 
I do not have anything that makes sense at the moment (Aside from checking if value origin is set but this seems like  a nasty hack)

Reply:
Well, I can live with it. Since we have the possibility to map more than 1 process to an enzyme having something like `CYP3A4 `and `CYP3A4 1` at the same time doesn't make much sense anymore. Thus as long as the values itself are properly read from the snapshot, I wouldn't invest time trying to set "changed" correctly for such a case